### ISSUE TITLE
[1.12] Projector displaying and placing belts wrong

### ISSUE THREAD

**Issue by ProxyPlayerHD** [Flaxbeard/ImmersivePetroleum#125]
After a confusing and long issue on the IE Github about the bottling machine not working we found out that it was actually the Projector being derpy with that one specific Multiblock.
Here is a link to the Issue which also explains further down what the exact problem with the Projector is:

https://github.com/BluSunrize/ImmersiveEngineering/issues/3147

it also states the versions of the mods, i so i hope this can be worked out and fixed.

**Comment by Flaxbeard** [comment]
Not surprising that the projector mechanism isn't perfect, hopefully an easy fix though

**Comment by ProxyPlayerHD** [comment]
that was 2 years ago...

**Comment by Flaxbeard** [comment]
Yes. I’ve been away from the modding community due to real life concerns.

**Comment by ProxyPlayerHD** [comment]
oh i see. well, welcome back and i hope it got better.
i mean, ignoring the current apocalypse.
